tests/plugins/stylescombo/manual/langattribute.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.21.1, feature, 5410
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, stylescombo
+
+
+1. Click on the Styles combo box to open the panel elements.
+2. Look at the dropped data.
+
+**Expected**
+
+* Item 0 should have the lang attribute set to `pl`.
+* Item 1 should have the `lang` attribute set to the editor language value.
```

Comment:
```suggestion
* Item 1 should have the lang attribute set to `pl`.
* Item 2 should have the `lang` attribute set to the editor language value.
```

Additionally, in my case the editor is `en-gb` but the second item is `en` 🤔 I don't know why as the code suggests they should be the same.

Reply:
Fixed. I added lang instead of language in the editor config.